api/README.md

```diff
@@ -0,0 +1,66 @@
+<img src="https://github.com/att/qujata/assets/7979328/199a9c84-840b-415e-a221-621c22184ad2" width="300">
+
+## Description
+This is a `Post Quantum Cryptography` tool,
+<br/>
+docker based with `Open Quantum Safe` images,
+<br/>
+using [Nest](https://github.com/nestjs/nest) framework TypeScript starter repository.
+
+## Platform limitations
+Pay attention that the Operating System of the machine running this project has an effect on the proper data returned in the results.
+<br/>
+Not all algorithms are equally well-supported on all platforms. In case of questions, it is first advised to review the [documentation files for each algorithm](https://github.com/open-quantum-safe/liboqs/tree/main/docs/algorithms).
+<br/>
+<ul>
+  <li>On Mac, it is expected to work well and was tested with AllowedAlgorithms.</li>
+  <li>On Windows, some Algorithms may not be supported by `Open Quantum Safe` docker images, but the code will work successfully.<br/>(Seeing `downloadSpeed=0` is suspected to be a sign of no-support for selected Algorithm).</li>
+  <li>Other Platforms were not tested yet.</li>
+</ul>
+
+## Pre-requisite
+[Node.js](#https://nodejs.org/en/download) (includes npm) and [Docker](https://docs.docker.com/engine/install) have to be installed on your system in order to run this project.
+
+## Installation
+1. To start, clone the qujata repository.
+```bash
+git clone https://github.com/att/qujata.git
+cd qujata/api
+```
+2. Install dependencies:
```

Comment:
we used '.' to end the line on 1 and ':' on 2
let's keep a same convention for both